### ISSUE TITLE
The osf.io website is too slow, takes minitues to load for simple webpages

### ISSUE THREAD

**Issue by yjiangnan** [CenterForOpenScience/osf.io#9330]
My location is inside China. The datacenter selected is in Australia. When I share my project in cell phone apps, it will take minutes to load the home wiki page. This will make it extremely difficult to share anything on OSF. Why does OSF has to be soooo sloooow compared to most other websites? It seems that a lot of elements need to be loaded sequentially before any real contents can be loaded.  

Additionally, when I open it, the " Project Wiki Pages" box takes the whole screen even though I only put two entries there, making people assume that there is no more content and abandon the page.

**Comment by adlius** [comment]
@yjiangnan Do you use a VPN service to connect to our website?

**Comment by yjiangnan** [comment]
Why not put a server inside China or somewhere close to China (such as Google cloud datacenter in Taiwan)? For users in China, it is known that there is the Great Fire Wall that slows everything down. The best choice is to put a server inside China. Otherwise, a server with CN2 GIA network is required for good Internet connections inside China.

**Comment by yjiangnan** [comment]
@adlius 
No. VPN only makes things worse for OSF. I connect to OSF directly.

**Comment by yjiangnan** [comment]
A server in Hong Kong may also improve the situation. But the software side may also need improvements.
